LeoProjectPayment/src/main/java/com/work/LeoProjectPayment/entity/Transaction.java

```diff
@@ -0,0 +1,46 @@
+package com.work.LeoProjectPayment.entity;
+
+import jakarta.persistence.Id;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Column;
+import lombok.Setter;
```

Comment:
Somethink is wrong with order of imports, there is an "organize imports" button in idea